src/Http/Headers/src/SameSiteMode.cs

```diff
@@ -3,10 +3,16 @@
 
 namespace Microsoft.Net.Http.Headers
 {
-    // RFC Draft: https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site-00
+    /// <summary>
+    /// Determines whether to send a cookies on "same-site" or "cross-site" requests
```

Comment:
Add a period at the end of this line. It'll get formatted as all in one line anyway (like HTML).

Reply:
Should the single enum value level comments get this same period?

Reply:
Good catch, yes they do, thanks for fixing.